client/homebrew/pages/basePages/listPage/brewItem/brewItem.jsx

```diff
@@ -104,6 +104,13 @@ const BrewItem = createClass({
 			</div>
 			<hr />
 			<div className='info'>
+				{brew.tags ? <>
+					<span title={`Tags:\n${brew.tags.join('\n')}`}>
```

Comment:
Could we also put  `className='tags'` onto this `<span/>` ?

Reply:
Makes sense, probably `className='brewTags'` so it won't clash with `className='systemTags'`, which will be coming Soon™ in a different PR.

Reply:
Should be in the latest commit now.